lib/configure.js

```diff
@@ -153,12 +153,12 @@ function configure (gyp, argv, callback) {
       return v
     }
 
-    log.silly('build/' + configFilename, config)
+    log.silly(gyp.buildDir + '/' + configFilename, config)
 
     // now write out the config.gypi file to the build/ dir
     var prefix = '# Do not edit. File was generated by node-gyp\'s "configure" step'
       , json = JSON.stringify(config, boolsToString, 2)
-    log.verbose('build/' + configFilename, 'writing out config file: %s', configPath)
+    log.verbose(gyp.buildDir + '/' + configFilename, 'writing out config file: %s', configPath)
```

Comment:
Can you wrap this line at 80 columns?  I'd break at the the comma, then line up the second argument with the first one.